libcloud/test/compute/fixtures/digitalocean/ex_enable_ipv6.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Since your change only applies to the v2 driver, this file is not needed right? Can you please remove it? Thanks!

Reply:
yup, done!

Reply:
@pquentin also see https://github.com/apache/libcloud/pull/1131, those two fixtures also are not needed I think because those methods only apply to the v2 driver as well